### PR TITLE
Added a banner

### PR DESCRIPTION
**Pull request by noraacode**
I took on a design job and created the website banner. The sizing was a bit off, but I adjusted the image directly in Canva and fixed that,

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,7 +69,7 @@
             height="315px"
             width="430px"
              />
-            10/22/2023: 9:00 AM-12:00 PM EST
+            11/05/2023: 10:00 AM-12:00 PM EST
             </div>
             <div class="announcements-right">
               <img
```
Comment (duongand) [inline index.html]: Add a class name here and then apply the following css to that class name
`
.className {
  height: 100%;
  widht: 100%;
}
`

File: styles.css
Comment (duongand) [inline styles.css:1]: Line 166 change value to 60px.

File: index.html
Comment (duongand) [inline index.html:1]: Images at line 93 and 104 need class names to then apply the following css rules

`.className { height: 75%; width: 90%; }`

**Review by duongand — CHANGES_REQUESTED** [review]
See comments

File: styles.css
Comment (noraacode) [reply styles.css:1]: line 166 is not line 166 for me. come to room #1 upstairs (that's my office) because I need personal assistance. thank you

**Comment by duongand** [comment]
LGTM

**Pull request merged**